src/main/java/com/qaprosoft/carina/demo/gui/components/GlossaryItem.java

```diff
@@ -32,27 +33,29 @@ public GlossaryItem(WebDriver driver, SearchContext searchContext) {
     }
 
     public boolean isHeaderEqualsListSize() {
-        LOGGER.info(nameParagraph.size() + " " + contentParagraph.size());
-        return (nameParagraph.size() == contentParagraph.size());
+        return (nameParagraphs.size() == contentParagraphs.size());
     }
 
     public boolean areParagraphMatchText() {
         boolean result = false;
-        for (int i = 1; i < nameParagraph.size(); i++) {
-            ExtendedWebElement tempParagraph = nameParagraph.get(i);
-            LOGGER.info(contentParagraph.get(i).getText());
-            List<String> tempText = Collections.singletonList(contentParagraph.get(i).getText());
-            LOGGER.info(String.valueOf(tempParagraph.getText().charAt(0)));
-            result = tempText.stream().anyMatch(t -> t.startsWith(String.valueOf(tempParagraph.getText().charAt(0))));
+        for (int i = 0; i < nameParagraphs.size(); i++) {
+            if (!Character.isDigit(nameParagraphs.get(i).getText().charAt(0))) {
+                ExtendedWebElement tempParagraph = nameParagraphs.get(i);
+                List<String> tempText = contentParagraphs.get(i).listOfContent();
+                result = tempText.stream().allMatch(t -> t.startsWith(String.valueOf(tempParagraph.getText().charAt(0))));
```

Comment:
I think you can update this
Example 
t.startsWith(tempParagraph.getText())
try it